inventories/local/group_vars/avalanche_nodes.yml

```diff
@@ -1,14 +1,13 @@
 # Copyright (C) 2022, Gauthier Leonard
 # See the file LICENSE for licensing terms.
 ---
-avalanchego_version: 1.9.0
+avalanchego_version: 1.9.6
 avalanchego_network_id: local
 avalanchego_use_existing_certs: yes
 avalanchego_local_certs_dir: "{{ inventory_dir }}/../../files/certs"
 
 avalanchego_vms_install:
-  - subnetevm=0.4.0
-  - spacesvm=0.0.9
+  - subnetevm=0.4.8
```

Comment:
For the Collection docs to work, we should start off with an empty list here. Otherwise the version compatibility check will fail:
```
TASK [ash.avalanche.node : Check that subnetevm=0.4.8 is compatible with avalanchego=1.9.9] ***
ok: [validator01] => (item={'key': 'ge', 'value': '1.9.6'}) => 
  msg: 'AvalancheGo version ge 1.9.6: True'
failed: [validator01] (item={'key': 'le', 'value': '1.9.8'}) => 
  msg: 'AvalancheGo version le 1.9.8: False'
fatal: [validator01]: FAILED! => 
  msg: One or more items failed
```